docs/developers/0_preparation.md

```diff
@@ -0,0 +1,24 @@
+# 💼 Preparing a developer environment
+In order to compile ReVanced Manager, certain requirements must be met.
+
+## 📝 Prerequisites
+* Kotlin IDE such as [Android Studio](https://developer.android.com/studio)
+* Understanding of [Android](https://android.com) development, [Kotlin](https://kotlinlang.org/) language and [Jetpack Compose](https://developer.android.com/jetpack/compose).
```

Comment:
Understanding of **Android development** do sound like you need to be expert in Android (OS) and not (App)